src/overlays/actors/ovl_En_Si/z_en_si.h

```diff
@@ -5,6 +5,8 @@
 
 #define ENSI_GET_CHEST_FLAG(thisx) (((thisx)->params & 0xFC) >> 2)
 
+#define SI_SPIDER_HOUSE_TOKENS_REQUIRED 30
```

Comment:
```suggestion
#define SPIDER_HOUSE_TOKENS_REQUIRED 30
```
This is probably fine without the prefix? Seems more like a global value, idk thoughts?